DatadogRUM/Sources/Feature/RUMFeature.swift

```diff
@@ -159,7 +159,7 @@ extension RUMFeature: Flushable {
     ///
     /// **blocks the caller thread**
     func flush() {
-        monitor.flush()
+        instrumentation.appHangs?.flush()
```

Comment:
Rationale: `monitor.flush()` is deleted in this PR. The "flush" for App Hangs instrumentation should had been added during App Hangs monitoring work.